e2e/various-suite/keybinds.spec.ts

```diff
@@ -37,14 +33,15 @@ describe('Keyboard shortcuts', options, () => {
       zone: 'Browser',
     });
     e2e.components.TimePicker.fromField().should('not.exist');
```

Comment:
I think we can get rid of these `e2e.components.TimePicker.fromField().should('not.exist');` lines - that was supposed to be what waits for the time picker to close, but i think waiting for the run button to be ready is good enough.